front/client/App.vue

```diff
@@ -3,7 +3,9 @@
     <toolbar></toolbar>
     <v-content>
       <v-container fluid>
-        <router-view></router-view>
+        <keep-alive exclude="article-page">
```

Comment:
Why do you need keep-alive for almost all the app, except article-page?

Reply:
there is an issue about cache whick I was not able to solve even with keep-alive. Please take a look